### PR TITLE
Feature/fix write method

### PR DESCRIPTION
**Pull request by kimberlythegeek**
building off of #144 by @matt-l-w 

fixed mock import error

### CODE REVIEW

**Comment by kimberlythegeek** [comment]
@matt-l-w r?

File: axe_selenium_python/tests/test_axe.py [hunk 1]
```diff
@@ -62,3 +63,31 @@ def _perform_axe_run(driver):
...
+    new_dir = tempfile.mkdtemp()
+    yield new_dir
+    shutil.rmtree(new_dir)
+
+
+def test_write_results_to_file(tempdir):
+    try:
+        from unittest.mock import MagicMock
+    except ImportError:
+        from mock import MagicMock
```
[the remark this answers is not in the document]
Comment (kimberlythegeek) [reply axe_selenium_python/tests/test_axe.py]: thanks!

File: axe_selenium_python/tests/test_axe.py [hunk 2]
```diff
@@ -62,3 +63,31 @@ def _perform_axe_run(driver):
     axe.inject()
     data = axe.execute()
     return data
+
+
+@pytest.fixture
+def tempdir():
```
[the remark this answers is not in the document]
Comment (kimberlythegeek) [reply axe_selenium_python/tests/test_axe.py]: ++

File: axe_selenium_python/axe.py [hunk 3]
```diff
@@ -100,5 +100,8 @@ def write_results(self, data, name="results.json"):
         :param name: Name of file to be written to.
         :param output: JSON object.
         """
-        with open(name, "r", encoding="utf8") as f:
-            f.write(json.dumps(data, indent=4))
+        with open(name, "w", encoding="utf8") as f:
+            try:
+                f.write(unicode(json.dumps(data, indent=4)))
+            except NameError:
+                f.write(json.dumps(data, indent=4))
```
[the remark this answers is not in the document]
Comment (kimberlythegeek) [reply axe_selenium_python/axe.py]: I'm running into an error with py27 when using `json.dump`

`TypeError: write() argument 1 must be unicode, not str`

```
def test_write_results_to_file(tmpdir, mocker):
    axe = Axe(mocker.MagicMock())
    data = {"testKey": "testValue"}
    filename = path.join(str(tmpdir), "results.json")
    axe.write_results(data, filename)
```
```
def write_results(self, data, name="results.json"):
    """
    Write JSON to file with the specified name.
    :param name: Name of file to be written to.
    :param output: JSON object.
    """
    with open(name, "w", encoding="utf8") as f:
        json.dump(data, f)
```

[the remark this answers is not in the document]
Comment (davehunt) [reply axe_selenium_python/axe.py]: Ugh, sorry - that sucks. You could consider using futures or six to make this code look a little nicer, but I'm not sure if it's worth the additional dependency.

**Pull request merged**